src/main/java/step2/domain/Ladder.java

```diff
@@ -0,0 +1,29 @@
+package step2.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Ladder {
+    public static final String HEIGTH_EXCEPTION = "사다리의 높이는 1이상이어야 합니다.";
+    private final int height;
+    public final List<Line> lines;
```

Comment:
```suggestion
    public static final String HEIGTH_EXCEPTION = "사다리의 높이는 1이상이어야 합니다.";

    private final int height;
    public final List<Line> lines;
```

상수와 인스턴스 변수 사이에는 한 줄을 띄워 주는 것이 좋습니다.